tests/unit/plugins/inventory/test_cobbler.py

```diff
@@ -16,11 +16,6 @@ def inventory():
     return InventoryModule()
 
 
-def test_init_cache(inventory):
-    inventory._init_cache()
-    assert inventory._cache[inventory.cache_key] == {}
```

Comment:
This test doesn't look very useful, and will stop working with how inventory plugins change for data tagging. So I've decided to simply remove it.